### PR TITLE
Fix generation of resource handlers for OpenShift

### PR DESCRIPTION
**Pull request by Vlatombe**
Broken since https://github.com/fabric8io/kubernetes-client/commit/25da8ca40ab2d23617181c1f74b938c6a757e5eb

### CODE REVIEW

**Comment by rohanKanojia** [comment]
ok to test

**Comment by Vlatombe** [comment]
@rohanKanojia done

File: openshift-client/src/test/java/io/fabric8/openshift/client/Handlersv4Test.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package io.fabric8.openshift.client;
+
```
Comment (rohanKanojia) [inline openshift-client/src/test/java/io/fabric8/openshift/client/Handlersv4Test.java]: Could you please add a license header here? This command would add it automatically:
```
mvn -N license:format
```

**Comment by Vlatombe** [comment]
@rohanKanojia Done.

**Comment by rohanKanojia** [comment]
retest this please

File: openshift-client/src/test/java/io/fabric8/openshift/client/Handlersv4Test.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import io.fabric8.openshift.api.model.v4_0.DeploymentConfig;
+import io.fabric8.openshift.api.model.v4_0.Group;
+import io.fabric8.openshift.api.model.v4_0.ImageStream;
+import io.fabric8.openshift.api.model.v4_0.ImageStreamTag;
+import io.fabric8.openshift.api.model.v4_0.OAuthAccessToken;
+import io.fabric8.openshift.api.model.v4_0.OAuthAuthorizeToken;
+import io.fabric8.openshift.api.model.v4_0.OAuthClient;
+import io.fabric8.openshift.api.model.v4_0.PolicyBinding;
+import io.fabric8.openshift.api.model.v4_0.Project;
+import io.fabric8.openshift.api.model.v4_0.RoleBinding;
```
Comment (rohanKanojia) [inline openshift-client/src/test/java/io/fabric8/openshift/client/Handlersv4Test.java]: Are you sure about this? I don't see any changes done in model with respect to this change. Could you please elaborate intention of having separate model for 4.0 openshift?

**Comment by Vlatombe** [comment]
@rohanKanojia Fixed. I guess I got confused because these imports showed up in the classpath in IDEA but I missed that it was not available in `openshift-client`. Anyway the first test is fine.

**Comment by rohanKanojia** [comment]
@Vlatombe : Could you please update CHANGELOG also? we released 4.5.2 last weekend.

**Comment by rohanKanojia** [comment]
We are good to merge this then :smile:

**Comment by Vlatombe** [comment]
@rohanKanojia You should consider using [Release Drafter](https://github.com/apps/release-drafter) for changelog, it's really good. Under the @jenkinsci organization we've been using it for a while

**Comment by rohanKanojia** [comment]
@Vlatombe : oh, cool. @oscerd @iocanel Wdyt?

**Comment by rohanKanojia** [comment]
[merge]

**Comment by iocanel** [comment]
@Vlatombe It looks good to me.

The only thing I would check, is that the changes done in the velocity template are algined with the rest of the modules. Its really handy if these variables are named the same across all templates, or we are going to have regressions next time we copy/paste something between them.

**Comment by Vlatombe** [comment]
@iocanel yes, I aligned the variables with the corresponding kubernetes templates.

**Pull request merged**